tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -460,7 +460,7 @@ class CompletionSuite extends BaseCompletionSuite {
   )
 
   check(
-    "commit1",
+    "commit1".tag(IgnoreScalaVersion("0.27.0-RC1")),
```

Comment:
Not sure why, but in 0.27.0-RC1 completions stopped working from things in the standard library.

Reply:
I will follow up in dotty and ask.